packages/transactions/tests/builder.test.ts

```diff
@@ -267,6 +267,90 @@ test('Make Multi-Sig STX token transfer', async () => {
   expect(serializedSignedTx.toString('hex')).toBe(signedTx);
 });
 
+test('Make Multi-Sig STX token transfer with two transaction signers', async () => {
+  const recipient = standardPrincipalCV('SP3FGQ8Z7JY9BWYZ5WM53E0M9NK7WHJF0691NZ159');
+  const amount = 2500000;
+  const fee = 0;
+  const nonce = 0;
+  const memo = 'test memo';
+
+  const authType = AuthType.Standard;
+  const addressHashMode = AddressHashMode.SerializeP2SH;
+
+  const privKeyStrings = [
+    '6d430bb91222408e7706c9001cfaeb91b08c2be6d5ac95779ab52c6b431950e001',
+    '2a584d899fed1d24e26b524f202763c8ab30260167429f157f1c119f550fa6af01',
+    'd5200dee706ee53ae98a03fba6cf4fdcc5084c30cfa9e1b3462dcdeaa3e0f1d201',
+  ];
+  const privKeys = privKeyStrings.map(createStacksPrivateKey);
+
+  const pubKeys = privKeyStrings.map(pubKeyfromPrivKey);
+  const pubKeyStrings = pubKeys.map(publicKeyToString);
+
+  const transaction = await makeUnsignedSTXTokenTransfer({
+    recipient,
+    amount,
+    fee,
+    nonce,
+    memo: memo,
+    numSignatures: 2,
+    publicKeys: pubKeyStrings,
+    anchorMode: AnchorMode.Any
+  });
+
+  const serializedTx = transaction.serialize();
+
+  const tx =
```

Comment:
I'm curious why we need to test the form of the serialization? is this in case we change the serialization algorithm? It seems to me enough to test the recovered values.

Reply:
It is generally pretty helpful to have these kinds of explicit vector tests. The benefits are basically:

1. They provide vectors to test other implementations. Imagine a hardware wallet implementation that can't use the TS/JS builds.
2. The tests become something of a canary for non-backwards-compatible changes in the serialization format.